templates/collection.liquid

```diff
@@ -1,9 +1,93 @@
+<section id="collection">
+  <div class="container">
+    <div class="grid-row">
+
+{% paginate collection.products by 24 %}
+
 {% if collection.all_products_count > 0 %}
-{% for product in collection.products %}
-<h2><a href="{{ product.url }}">{{ product.title }}</a></h2>
-{{ product.description }}
-<img src="{{ product.featured_image | product_img_url: 'grande' }}" alt="{{ product.title | escape }}" />
-{% endfor %}
+
+  {% if collection.url != blank %}
+
+      <div class="grid-item item-s-12 item-m-auto grid-row no-gutter align-content-start">
```

Comment:
I think this isn't such a good approach as it already causes layout issues:
![screen shot 2017-10-17 at 14 01 06](https://user-images.githubusercontent.com/845186/31666334-bcc2503c-b343-11e7-81ea-76b197648f2b.png)

Also I think it isn't a good because it breaks the grid which in my opinion breaks the design.

Reply:
(that image is a `m` width but before the 2 products fit on the right of the side menu)

Reply:
its to maximize space for the products

Reply:
you're seeing it wrong. you need to push the correct branch

Reply:
setup a testing theme for yourself

Reply:
oh you did. well what i'm seeing in this screengrab doesn't correspond to the branch.  you should `theme upload` from the branch probably and activate your theme.

Reply:
should look like this

<img width="1279" alt="screen shot 2017-10-17 at 10 53 27 am" src="https://user-images.githubusercontent.com/7727722/31675178-778b0c9e-b329-11e7-9072-0fcfcb5cb1da.png">